src/commands/ref_command_handler.js

```diff
@@ -34,16 +33,24 @@ function cppReference(message, args) {
                 }
             }
 
-
             const results = [];
             for (const ref of hrefs) {
                 if (ref.search(args[0]) > 0) {
                     console.log(ref);
                     results.push("https://en.cppreference.com/" + ref);
                 }
             }
+			
             if (results.length > 0) {
-                channel.send(results);
+				var l = 0;
+				for (var i = 0; i < results.length; i++)
```

Comment:
This could replaced by a `for of` loop.

Also please don't use `var` but use `const` or `let` instead. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#Scoping_rules_2

Also please use { } for ifs and fors, even if it is a one-liner.